requirements/runtime.txt

```diff
@@ -1,5 +1,4 @@
 # Package runtime requirements.
-convertdate>=2.3.0
 hijri-converter
```

Comment:
Now we can remove `hijri-converter` too
```suggestion
```

Reply:
Yeah, I guess we can do it either separately or in more related PR, e.g., https://github.com/dr-prodigy/python-holidays/pull/1128